Vagrantfile

```diff
@@ -1,16 +1,23 @@
 Vagrant.configure("2") do |config|
-  # We use this image to get a large (160GB) disk rather than the normal 40GB.
-  config.vm.box = "cbednarski/ubuntu-1604-large"
-  config.vm.box_version = "0.1.0"
+  config.vm.box = "generic/ubuntu1804"
+  config.vm.box_version = "1.9.34"
 
   config.vm.provision :shell, privileged: false, path: "infrastructure/indexer-provision.sh"
   config.vm.provision :shell, privileged: false, path: "infrastructure/vagrant/indexer-provision.sh"
   config.vm.provision :shell, privileged: false, path: "infrastructure/web-server-provision.sh"
 
-  config.vm.network :forwarded_port, guest: 80, host: 8000
+  config.vm.network :forwarded_port, guest: 80, host: 8001
```

Comment:
The port was because it conflicts with WPT and was doing something else, can revert or update the docs.

Reply:
+1 to this.  I filed https://bugzilla.mozilla.org/show_bug.cgi?id=1491956 on this previously (but didn't push a fix).